assets/eip-4675/contracts/math/SafeMath.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: ISC
```

Comment:
I am not a lawyer, but if this is copied from OpenZeppelin, I don't think you can just change the license.